src/main/java/calculator/InputString.java

```diff
@@ -0,0 +1,54 @@
+package calculator;
+
+import java.util.regex.Pattern;
+
+public class InputString {
+    private final String input;
+    private String body;
+    private String pattern;
+    private String[] tokens;
```

Comment:
```suggestion
public class InputString {
    private final String input;
    private String expression;
    private String operatorRegex;
    private String[] tokens;
```

`body`, `pattern` 말고 더 구체적인 이름을 사용해 보시면 어떨까요?

Reply:
구현할 당시에는 충분히 구체적이라고 생각했는데 지금 보니 아니네요.......... 네이밍에 조금 더 신경을 써야겠습니다. 감사합니다!